openedx/core/djangoapps/profile_images/images.py

```diff
@@ -90,25 +83,48 @@ def validate_uploaded_image(uploaded_file):
     # see also: http://en.wikipedia.org/wiki/Magic_number_%28programming%29
 
     if uploaded_file.size > settings.PROFILE_IMAGE_MAX_BYTES:
-        raise ImageValidationError(FILE_UPLOAD_TOO_LARGE)
+        too_large_message = _(
+            u'The file must be smaller than {image_max_size} in size.'.format(
```

Comment:
Pylint will tell you this, but you need to use only a string literal in `_()`.  It has to be translate first, and then format.